lib/sms.rb

```diff
@@ -0,0 +1,15 @@
+require "bundler"
+Bundler.require()
+
+class Sms
+  account_sid = ENV['TWILIO_ACCOUNT_SID']
+  auth_token = ENV['TWILIO_AUTH_TOKEN']
+
+  @client = Twilio::REST::Client.new(account_sid, auth_token)
+
+  @client.messages.create(
+    to: ENV['MY_PHONE_NUMBER'],
+    from: ENV['TWILIO_PHONE_NUMBER'],
+    body: "Thank you! Your order was placed and will be delivered within one hour."
+  )
+end
```

Comment:
Using the Twilio API was the tricky part of this challenge.  I'd recommend that you take a look at the exemplar for this file: https://github.com/makersacademy/takeaway-exemplar/blob/master/lib/sms.rb